src/rpcgen/trn_rpc_protocol.x

```diff
@@ -224,6 +224,53 @@ struct rpc_trn_vsip_ppo_key_t {
        uint16_t port;
 };
 
+/* Defines a pod label policy table entry */
+struct rpc_trn_pod_label_policy_t {
+       string interface<20>;
+       uint64_t tunid;
+       uint32_t pod_label_value;
+       uint64_t bit_val;
+};
+
+/* Defines a pod label policy table key */
+struct rpc_trn_pod_label_policy_key_t {
+       string interface<20>;
+       uint64_t tunid;
+       uint32_t pod_label_value;
+};
+
+/* Defines a namespace label policy table entry */
+struct rpc_trn_namespace_label_policy_t {
+       string interface<20>;
+       uint64_t tunid;
+       uint32_t namespace_label_value;
+       uint64_t bit_val;
+};
+
+/* Defines a namespace label policy table key */
+struct rpc_trn_namespace_label_policy_key_t {
+       string interface<20>;
+       uint64_t tunid;
+       uint32_t namespace_label_value;
+};
+
+/* Defines a pod and namespace label policy table entry */
+struct rpc_trn_pod_and_namespace_label_policy_t {
+       string interface<20>;
+       uint64_t tunid;
+       uint32_t pod_label_value;
+       uint32_t namespace_label_value;
```

Comment:
Does 0 here mean this is equivalent to rpc_trn_pod_label_policy_t?

Reply:
In "pod and namespace", there won't have 0.